storybookjs/design-system
# Pulled in `AddonItem` component from `frontpage`

Fixes https://linear.app/chromaui/issue/CH-734/pull-addonitem-from-frontpage-into-ds

## Description

Pulled the `AddonItem` component from https://github.com/storybookjs/frontpage into the DS in preparation for future work and called it `DisplayCard` instead as it will be used for more general purposes.

Also pulled in the `VerifiedBadge` component from https://github.com/storybookjs/frontpage as it's used in the `AddonItem`.

### QA instructions

Confirm the styles (mostly) match [the designs]((https://www.figma.com/file/gYdEmX9zYowUR3K2SOua5J/Component-catalog?node-id=180%3A8141).

<details>
  <summary>📦 Published PR as canary version: <code>7.0.3-canary.320.c1268bc.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/design-system@7.0.3-canary.320.c1268bc.0
  # or 
  yarn add @storybook/design-system@7.0.3-canary.320.c1268bc.0
  ```
</details>
